e2e/flags.go

```diff
@@ -250,6 +281,8 @@ const (
 	GitLab = "gitlab"
 	// CSR indicates using Google Cloud Source Repositories to host the repositories.
 	CSR = "csr"
+	// ArtifactRegistry indicates using Google Artifact Registry to host the repositories.
+	ArtifactRegistry = "artifact_registry"
```

Comment:
If we have to type this out sometimes, i suggest "gar" as the typical acronym, similar to "csr".

Or we could spell out csr. If we spell it out i guess I would recommend hyphens, rather than underscores, but I guess it's not really a big deal.

Reply:
Done